docs/pages/modules/module-api.mdx

```diff
@@ -360,6 +360,16 @@ Some common iOS types from `CoreGraphics` and `UIKit` system frameworks are alre
 | `CGRect`                | `{ x: number, y: number, width: number, height: number }` or `number[]` with _x_, _y_, _width_ and _height_ values                                                                |
 | `CGColor`<br/>`UIColor` | Color hex strings (`#RRGGBB`, `#RRGGBBAA`, `#RGB`, `#RGBA`), named colors following the [CSS3/SVG specification](https://www.w3.org/TR/css-color-3/#svg-color) or `"transparent"` |
 
+Similarly, some common Android types from packages like `java.io`, `java.net`, or `android.graphics` are also made convertible.
+
+| Native Android Type                     | TypeScript                                                                                                                                                                         |
+| --------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `java.net.URL`                          | `string` with a URL. Note that the scheme has to be provided.                                                                                                                      |
+| `android.net.Uri`<br/>`java.net.URI`    | `string` with a URI. Note that the scheme has to be provided.                                                                                                                      |
+| `java.io.File`<br/>`java.nio.file.Path` | `string` with a path to the file.                                                                                                                                                  |
+| `android.graphics.Color`                | Color hex strings (`#RRGGBB`, `#RRGGBBAA`, `#RGB`, `#RGBA`), named colors following the [CSS3/SVG specification](https://www.w3.org/TR/css-color-3/#svg-color) or `"transparent"`. |
+| `kotlin.Pair<A, B>`                     | Array with two values, where the first one is of type _A_ and the second is of type _B_.                                                                                           |
```

Comment:
```suggestion
| `java.net.URL`                          | `string` with a URL. Note that the scheme has to be provided                                                                                                                      |
| `android.net.Uri`<br/>`java.net.URI`    | `string` with a URI. Note that the scheme has to be provided                                                                                                                      |
| `java.io.File`<br/>`java.nio.file.Path` | `string` with a path to the file                                                                                                                                                  |
| `android.graphics.Color`                | Color hex strings (`#RRGGBB`, `#RRGGBBAA`, `#RGB`, `#RGBA`), named colors following the [CSS3/SVG specification](https://www.w3.org/TR/css-color-3/#svg-color) or `"transparent"` |
| `kotlin.Pair<A, B>`                     | Array with two values, where the first one is of type _A_ and the second is of type _B_                                                                                           |
```
Just to follow the pattern of the previous (iOS) table, I think we can leave the period from items in this table too for consistency.